### PR TITLE
Feature/add teacher to class

### PR DESCRIPTION
**Pull request by secretarius**
create connection teacher to journal from the teacher-list component

### CODE REVIEW

File: src/app/admin-panel/teacher-connection/snackbar-mess/snackbar-mess.component.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ .choise-done {
+      color: white;
```
Comment (dmitryy90) [inline src/app/admin-panel/teacher-connection/snackbar-mess/snackbar-mess.component.scss]: fix indentations

File: src/app/admin-panel/teacher-connection/snackbar-mess/snackbar-mess.component.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<span class="choise-done">
+  Вибір зроблено!!!
```
Comment (dmitryy90) [inline src/app/admin-panel/teacher-connection/snackbar-mess/snackbar-mess.component.html]: please, clarify wording that is used across the application

File: src/app/admin-panel/teacher-connection/snackbar-mess/snackbar-mess.component.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-snackbar-mess',
```
Comment (dmitryy90) [inline src/app/admin-panel/teacher-connection/snackbar-mess/snackbar-mess.component.ts]: you can try to make one file component

File: src/app/services/teachers-journal.service.spec.ts [hunk 4]
```diff
@@ -1,16 +1,55 @@
...
+};
+
+const mockPost = {
+  class: {},
+  subject: {},
+  teacher: {}
+};
+
 describe('TeachersJournalService', () => {
+<<<<<<< HEAD
```
Comment (dmitryy90) [inline src/app/services/teachers-journal.service.spec.ts]: conflict marker

File: src/app/admin-panel/teacher-connection/snackbar-mess/snackbar-mess.component.scss (hunk 1 above)
Comment (secretarius) [reply src/app/admin-panel/teacher-connection/snackbar-mess/snackbar-mess.component.scss]: resolved

File: src/app/admin-panel/teacher-connection/snackbar-mess/snackbar-mess.component.html (hunk 2 above)
Comment (secretarius) [reply src/app/admin-panel/teacher-connection/snackbar-mess/snackbar-mess.component.html]: resolved

File: src/app/admin-panel/teacher-connection/snackbar-mess/snackbar-mess.component.ts (hunk 3 above)
Comment (secretarius) [reply src/app/admin-panel/teacher-connection/snackbar-mess/snackbar-mess.component.ts]: resolved

File: src/app/services/teachers-journal.service.spec.ts (hunk 4 above)
Comment (secretarius) [reply src/app/services/teachers-journal.service.spec.ts]: resolved

**Pull request merged**